.gitignore

```diff
@@ -22,8 +22,8 @@ tests/coverage.info
 /target/
 .tmp_loader.json
 build/
-src/main/java/com/intel/genomicsdb/GenomicsDBCallsetsMapProto.java
-src/main/java/com/intel/genomicsdb/GenomicsDBExportConfiguration.java
-src/main/java/com/intel/genomicsdb/GenomicsDBImportConfiguration.java
-src/main/java/com/intel/genomicsdb/GenomicsDBVidMapProto.java
```

Comment:
Probably a good idea to keep these old listings as well

Reply:
Ok, I can add those back.